include/cuco/detail/reduction_ops.cuh

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
This file shouldn't be in `detail/` since users are expected to use these types when instantiating a `static_reduction_map`.

Reply:
[c2e4e62](https://github.com/NVIDIA/cuCollections/pull/98/commits/c2e4e6212ee612a05ef873b9d2b988efa5d4e560)